README.md

```diff
@@ -1,3 +1,27 @@
 # Cast
 
 A podcast listening PWA
+
```

Comment:
[nitpick (non-blocking)](https://conventionalcomments.org/#labels): I don't think we really want this copy-paste in our README :)

Reply:
Updated with the standard `pnpm` commands.